packages/legend-graph/src/graph-manager/protocol/pure/v1/model/packageableElements/store/relational/model/V1_TablePtr.ts

```diff
@@ -33,3 +34,12 @@ export class V1_TablePtr implements Hashable {
     ]);
   }
 }
+
+export const V1_buildTablePtr = (protocol: V1_TablePtr): TablePtr => {
```

Comment:
lets keep the protocol models clean. we should have this in the relational helper builder